lib/railway_ipc/message_consumption.ex

```diff
@@ -71,6 +82,26 @@ defmodule RailwayIpc.MessageConsumption do
     {:error, message_consumption}
   end
 
+  def handle_message(:no_uuid, %{
+    handle_module: handle_module,
+    inbound_message: %{decoded_message: message}
+  }) do
+    case handle_module.handle_in(message) do
+      :ok ->
+        metadata = [
+          feature: "ipc_republish_message",
```

Comment:
Same issue as my comment on https://github.com/learn-co/railway_ipc/pull/15/files#diff-ff77afc03658fe715548deb63c0556e2R55

Reply:
I removed this code